R/precipitation_surplus.R

```diff
@@ -120,3 +121,47 @@ calc_psp <- function(B_LU_BRP, M_GREEN){
   
 }
 
+
+
+#' Calculate indicator for precipitation surplus
+#' 
+#' This function calculates the indicator value for precipitation surplus
+#' 
+#' @param D_PSP (numeric) The precipitation surplus per crop  calculated by \code{\link{calc_psp}}
+#' @param B_LU_BRP (numeric) The crop code from the BRP 
+#'     
+#' @export
+ind_psp <- function(D_PSP,B_LU_BRP){
+  
+  crops.obic = crop_name = crop_code = I_PSP = NULL
+  
+  # Check input
+  arg.length <- max(length(D_PSP), length(B_LU_BRP))
+  
+  checkmate::assert_numeric(D_PSP, any.missing = FALSE, len = arg.length)
+  checkmate::assert_numeric(B_LU_BRP, any.missing = FALSE, min.len = 1, len = arg.length)
+  checkmate::assert_subset(B_LU_BRP, choices = unique(OBIC::crops.obic$crop_code), empty.ok = FALSE)
+
+  dt <- data.table(D_PSP,
+                   B_LU_BRP)
+  
+  # Separate evaluation for grassland if sep_eval = TRUE
+  sep_eval <- FALSE
+  
+  if(sep_eval){
```

Comment:
If `sep_eval` is always `FALSE`, as defined in L149, why is this condition then still needed?

Reply:
@gerardhros?